### DISCUSSION TITLE
`git-town sync` and `rebase.updateRefs`

### DISCUSSION THREAD

**Discussion by sergej-koscejev** [git-town/git-town#4101] [Q&A] [answered]
It seems that `git-town sync` does not work correctly when [rebase.updateRefs](https://git-scm.com/docs/git-rebase#Documentation/git-rebase.txt---update-refs) is enabled. Is that correct?

**Comment by kevgo** [comment, marked as answer]
Thanks for reporting this. I have fixed this in https://github.com/git-town/git-town/pull/4102. This will be released in a new version of Git Town within the next couple of days.

**Reply by sergej-koscejev** [reply]
Wow, that was quick, thank you!

**Reply by kevgo** [reply]
This is now shipped as part of [Git Town 16.4.1](https://github.com/git-town/git-town/releases/tag/v16.4.1)!
